### PR TITLE
Prisoner Role

### PR DESCRIPTION
**Pull request by Geevies**
## About The Pull Request

Implements roundstart USCM Prisoners and PoWs. There is an 80% chance for you to just be your PFC character that got arrested for the entire op for messing around some time before the op, but there's a 15% chance that you're a CLF Guerilla and a 5% chance that you're a CLF Cell Leader, with a randomized CLF name. Lorewise, the PFC is being held in custody until after the op, as they're untrustworthy. As for the CLF, they were captured during Operation Tachyon Tackle and the Almayer is holding them until they can be offloaded at a dock.

## Why It's Good For The Game

It's cool. Gives MPs something to do when marines aren't being antags, gives people a role that lets them keep shipside entertained. As well as being spicy when an ERT decides to do a prisonbreak.

## Changelog

:cl:
add: Added prisoner role, with a 80% chance of being a PFC, a 15% chance of being a CLF guerilla, and a 5% chance of being a CLF cell leader.
/:cl:

### CODE REVIEW

**Comment by Drevenshek** [comment]
i would make it so their all CLF prisoners tbh would make it more interesting

**Comment by stanalbatross** [comment]
@Drevenshek feedback goes in discord now

File: code/modules/gear_presets/uscm_ship.dm [hunk 1]
```diff
@@ -669,3 +669,99 @@
...
+	name = "USCM Prisoner (PFC)"
+	flags = EQUIPMENT_PRESET_START_OF_ROUND
+	var/preset_type = /datum/equipment_preset/uscm/pfc
+	var/datum/equipment_preset/preset
+
+	var/entry_message = {"You were arrested for %ARRESTREASON%. As such, you're being held inside the brig until after the current operation finishes."}
+	var/list/arrest_reasons = list(
+		"extreme hooliganism",
+		"pantsing the CO",
+		"converting the APC's catalytic converter into a bong",
```
Comment (Agameofscones) [inline code/modules/gear_presets/uscm_ship.dm:685]: nice

**Comment by cm13live** [comment]
This pull request has been mentioned on **CM-SS13**. Please provide feedback here:

https://forum.cm-ss13.com/t/add-a-new-role-prisoner/1430/5

**Pull request closed without merging**